Makefile

```diff
@@ -84,6 +84,9 @@ $(CACHE)/.setup:
 		$(SRC_DIR)/randomGamePicker \
 		$(SRC_DIR)/easter \
 		-depth -type d -name res -exec cp -r {}/. $(BUILD_DIR)/.tmp_update/res/ \;
+	@find \
+	    $(SRC_DIR)/libgamename \
+		-depth -type d -name res -exec cp -r {}/. $(BUILD_DIR)/ \;
```

Comment:
Not a fan of this location

Reply:
This is the txt file containing the full list of games extracted from the various emulators. It is modifiable by the users if the want add games or change the titles to display. I thought in the root was quite evident to see and easily updatable, but if you have a better proposal where to put it I will change.

Reply:
See my other comment